eslint.config.mjs

```diff
@@ -0,0 +1,95 @@
+import typescriptEslint from '@typescript-eslint/eslint-plugin';
+import globals from 'globals';
+import tsParser from '@typescript-eslint/parser';
+import svelteParser from 'svelte-eslint-parser';
+import js from '@eslint/js';
+
+export default [
+	js.configs.recommended,
+	{
+		ignores: [
+			'**/node_modules/**',
+			'build/**',
+			'.svelte-kit/**',
+			'package/**',
+			'**/*.d.ts',
+			'.vercel/**'
+		]
+	},
+	{
+		files: ['**/*.{js,ts,svelte}'],
+		plugins: {
+			'@typescript-eslint': typescriptEslint
+		},
+		languageOptions: {
+			globals: {
+				...globals.browser,
+				...globals.node,
+				__VER__: 'readonly'
+			},
+			parser: tsParser,
+			parserOptions: {
+				ecmaVersion: 2020,
+				sourceType: 'module',
+				extraFileExtensions: ['.svelte']
+			}
+		},
+		rules: {
+			'@typescript-eslint/ban-ts-comment': 'error',
+			'@typescript-eslint/naming-convention': [
+				'error',
+				{
+					selector: 'default',
+					format: null
+				},
+				{
+					selector: 'variable',
+					format: ['snake_case'],
+					leadingUnderscore: 'allow',
+					trailingUnderscore: 'allow'
+				},
+				{
+					selector: 'variable',
+					modifiers: ['const'],
+					format: ['UPPER_CASE']
```

Comment:
How do we make it so function expressions aren't flagged? Right now:

```
export const form_action = () => { ... }
```

This is flagged as needing to be uppercase, which doesn't feel right.

Reply:
Fixed in [702ba90\